testing/bdd/features/handover/handover.feature

```diff
@@ -60,25 +82,36 @@ Feature: Handover algorithm
     And doc1 is added to the sequence
     And we create a new document
     And it has sequenceNumber 1
-    And it does not have element authorsGroupIdentifier
+    And it does not have attribute authorsGroupIdentifier
     Then doc2 is added to the sequence
```

Comment:
is doc2 the new document? Since it has the same `sequenceNumber` it will be ignored and not added to the sequence, regardless of `authorsGroupIdentifier`.